pkg/model/logstore/logstore.go

```diff
@@ -532,10 +568,41 @@ func (s *LogStore) startAndLastIndices(spans map[SpanID]*Span) (startIndex, last
 	return startIndex, lastIndex
 }
 
-func (s *LogStore) logHelper(spansToLog map[SpanID]*Span, showManifestPrefix bool) string {
+type logOptions struct {
+	spans                       map[SpanID]*Span
+	showManifestPrefix          bool
+	skipFirstLineManifestPrefix bool
+}
+
+func (s *LogStore) logHelper(options logOptions) string {
```

Comment:
nit: This is pre-existing but I don't think `logHelper` is a good name for this function. `logOptionsToString` maybe?

Reply:
i changed it to `toLogString`... didn't want to imply that we were converting the logOptions struct itself into a string